docs/output.md

```diff
@@ -465,6 +488,11 @@ Note that filtered FASTA is only used for BGC workflow for run-time optimisation
   - `<sample>/*_ampcombi.tsv`: summarised output in tsv format for each sample
   - `<sample>/*_amp.faa*`: fasta file containing the amino acid sequences for all AMP hits for each sample
   - `<sample>/*_mmseqs_matches.txt*`: alignment file generated by MMseqs2 for each sample
+
+:::info
+In some cases, when the AMP workflow is turned on, only summary files per sample will be created in the output folder with **NO** `Ampcombi_summary.tsv` and `Ampcombi_summary_cluster.tsv` files and hence no taxonomic classifications will be merged (if the taxonomic classification subworkflow is turned on). This can occur when strictly setting parameters which can lead to no AMP hits found in any of the samples or in only one sample. Look out for `[nf-core/funcscan] AMPCOMBI2: 0/1 file passed. Skipping AMPCOMBI2_COMPLETE, AMPCOMBI2_CLUSTER, and TAXONOMY MERGING steps.`in the stdout or `.nextflow.log` file.
```

Comment:
I don't understand exactly what you're trying to say here...

Is it: You will only get Ampcombi_summary/tsv and cluster.tsv if taxonomic classification is executed?

Or is it simply, if there are no XYZ hits you will not get those files?

Reply:
exactly your last interpretation :)

Reply:
I rephrased it a bit